lib/internal/assert/myers_diff.js

```diff
@@ -35,22 +34,29 @@ function myersDiff(actual, expected, checkCommaDisparity = false) {
     ArrayPrototypePush(trace, newTrace);
```

Comment:
I think we can remove the `newTrace` here
```suggestion
    ArrayPrototypePushApply(trace, v);
```